examples/animation/implicit/container7/lib/main.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:flutter_web/material.dart';
+import 'package:flutter_web_test/flutter_web_test.dart';
+import 'package:flutter_web_ui/ui.dart' as ui;
+
+const owl_url =
```

Comment:
nit: I would just leave this on one line (unless you are bound to 80 chars per line)